webpack.common.config.js

```diff
@@ -68,17 +69,22 @@ module.exports = {
     }),
 
     new webpack.DefinePlugin({
-      // This is a temporary fix for okta-auth-js not populating
-      // this when consumed via npm
-      SDK_VERSION: JSON.stringify(0),
-      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken'),
-      DEFAULT_POLLING_DELAY: 500,
-      FRAME_ID: JSON.stringify('okta-oauth-helper-frame')
+      WIDGET_VERSION: JSON.stringify(packageJson.version)
```

Comment:
unfortunately, yes. DefinePlugin is basically a find/replace, so when webpack tries to replace `WIDGET_VERSION` in the code, it'll do it without quotes. `JSON.stringify` adds quotes.